src/main/resources/hudson/plugins/jobConfigHistory/JobConfigHistory/config.jelly

```diff
@@ -1,54 +1,60 @@
 <?jelly escape-by-default='true'?>

```

Comment:
Side note: I think if the HTML help files are moved from `src/main/webapp/help/` to `src/main/resources/hudson/plugins/jobConfigHistory/JobConfigHistory/` which follows the convention than the `help` attributes can be removed and the Jelly file gets even simpler.

Reply:
The help message were still working and I thought I would just keep the changes to a minimum. 
Anyway, I have moved them now.